install.sh

```diff
@@ -651,84 +651,6 @@ do_install() {
 			echo_docker_as_nonroot
 			exit 0
 			;;
-		sles)
-			if [ "$(uname -m)" != "s390x" ]; then
```

Comment:
Could we also print some message with an explanation?

Is it okay to drop support for installing versions that were already released for SLES?

Reply:
> Is it okay to drop support for installing versions that were already released for SLES?

Yes we can drop support.

Reply:
Could we still print an error here saying that SLES packages are no longer supported?

Reply:
Sorry, I missed this PR!


Curious; is the plan to drop immediately, or with the next major release (docker v28)? Wondering if that would be an option, so that it can be more clearly communicated that "starting with v28, SLES on s390x is no longer supported" (or something along those lines).


> Could we still print an error here saying that SLES packages are no longer supported?

Yeah, probably could use a message added here; https://github.com/docker/docker-install/blob/24f7a3141dd6bd50ee46b019b0f4690119143627/install.sh#L469-L492

Reply:
We plan to drop support immediately. When the installation is tried on SLES and RHEL (for s390x), we can add a note rather than a deprecation notice. The note will mention "Effective v27.5, please consult SLES distro statement for s390x support."